src/test/java/org/folio/rest/impl/FeeFineReportsAPITest.java

```diff
@@ -544,6 +561,12 @@ private Feefineaction createAction(int actionCounter, Account account, String da
       staffInfo, patronInfo, txInfo);
   }
 
+  private Feefineaction createActionWithoutComments(Account account, String dateTime,
```

Comment:
I think something like `createActionWithNullComments` would be more explicit. `without` doesn't really tell much - is it empty string, null or maybe the field is missing.